google/cloud/bigquery/model.py

```diff
@@ -433,6 +447,66 @@ def __repr__(self):
             self.project, self.dataset_id, self.model_id
         )
 
+class TransformColumn:
+    """TransformColumn represents a transform column feature.
+
+    See
+    https://cloud.google.com/bigquery/docs/reference/rest/v2/models#transformcolumn
+
+    Args:
+        column_name:
+            The column name of the transform column.
+        column_type:
+            The column type of the transform column.
+        transform_sql:
+            The SQL expression of the transform column.
+    """
+    def __init__(self,resource: Dict[str, Any]):
+        self._properties = resource
+        self.name = resource.get("name")
+        type_json = resource.get("type")
+        if type_json is None:
+            self.type_ = None 
+        else:
+            self.type_ = standard_sql.StandardSqlDataType.from_api_repr(type_json)
+        self.transform_sql = resource.get("transformSql")
+        
+    @property
+    def name(self) -> str:
```

Comment:
Since we're using `get(...)`, it could return `None`. Return type `Optional[str]` instead of `str`.